docs/docs/working-with-entity-manager/5-entity-manager-api.md

```diff
@@ -62,6 +62,7 @@ const rawData = await manager.query(
 const rawData = await manager.query(
     "SELECT * FROM USERS WHERE name = @0 and age = @1",
     ["John", 24],
+    true, // optional flag to return a structured result set object containing all recordsets
```

Comment:
I don't think this is the right place to show this flag, we don't want to suggest that it should be used by default when querying mssql.

And the structured result feature also applies to other drivers, even if they don't all support multiple recordsets.

Suggest we add another paragraph describing the `query` options object just below this example.